test.py

```diff
@@ -90,6 +90,9 @@ def _import_from_file(script):
     os.path.join('domain', 'brain_observatory.py'),  # TODO create separate workflow for this
 ]
 
+deprecated_examples = [
```

Comment:
One question that came up during these deprecation changes was how to handle the "Intracellular Electrophysiology Data using SweepTable" tutorial. My initial thought was to skip the gallery example testing for this file (`docs/gallery/domain/icephys.py`), but otherwise leave the tutorial as is so that these examples are still accessible to anyone using older files or versions of pynwb.

Reply:
I think it would be confusing to people to have a tutorial that shows a deprecated way of doing things. I would be in favor of removing this. We could activate docs for older versions of PyNWB to make sure that information is still available.

Reply:
Having docs for older versions of PyNWB sounds like a much better solution! I will remove this tutorial then.